tests/api_fastapi/core_api/routes/public/test_auth.py

```diff
@@ -46,7 +46,7 @@ class TestGetLogin(TestAuthEndpoint):
         ],
     )
     def test_should_respond_307(self, test_client, params):
-        response = test_client.get("/public/auth/login", follow_redirects=False, params=params)
+        response = test_client.get("/api/v2/auth/login", follow_redirects=False, params=params)
```

Comment:
Simiarly on this one? Can/should we configure `test_client` etc so that

```suggestion
        response = test_client.get("auth/login", follow_redirects=False, params=params)
```

works?

Reply:
Yes it would be a nice factorization. Could be a follow up PR too so it doesn't add too much effort for this PR.

Reply:
See tracking issue #47977